shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -296,6 +297,8 @@ protected Bundle createArgs() {
   private FlutterView flutterView;
   @Nullable
   private PlatformPlugin platformPlugin;
+  @NonNull
+  private final PlatformViewsController platformViewsController = new PlatformViewsController();
```

Comment:
Are we moving the ownership of the platformviewscontroller out of the plugins registry into here?

Reply:
Yes. As mentioned in the doc for the new plugin registry, its job is now exclusively to register plugins, not to initialize and hold arbitrary resources and dependencies.

Reply:
Talked offline, resolving old comments.